src/lib/components/inputs/Text.svelte

```diff
@@ -44,12 +44,11 @@
     overflow: hidden;
     text-overflow: ellipsis;
     font-family: "Source Sans Pro";
-    font-size: 1rem;
     font-style: normal;
```

Comment:
I saw you removed the text size in another commit. This is needed to prevent the default browser `font-size` for inputs, which is tiny. I've added an override in case we need text inputs of different sizes (a title input sized `--font-xl`, for example).